Objects/Converters/ConverterRevit/ConverterRevitShared/PartialClasses/ConvertRebar.cs

```diff
@@ -61,27 +61,31 @@ public ApplicationObject RebarToNative(RevitRebarGroup speckleRebar)
       speckleRebar.shape.rebarType == RebarType.Standard ? RebarStyle.Standard : RebarStyle.StirrupTie;
 
     // get start and end hooks and orientations
-    var speckleStartHook = speckleRebar.startHook as RevitRebarHook;
     RebarHookType startHook = null;
     RebarHookOrientation startHookOrientation = RebarHookOrientation.Right;
-    if (speckleStartHook != null)
+    if (speckleRebar.startHook is RevitRebarHook speckleStartHook)
     {
       startHook = RebarHookToNative(speckleStartHook);
```

Comment:
We expect RebarHookToNative to return something but the if the TryParse() fails the startHook is left as .Right - is this sensible? Would we perhaps eschew ding the conversion if we cannot correctly classify the hook? I think maybe this is best as new Speckle data might have hooks older plugins cannot parse?

Reply:
It also begs the question as to how we see such micro failures - we _expect_ speckleStartHook.orientation, to be a type we understand but if it is not we don't do anything about it...